Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.cs

```diff
@@ -75,5 +79,31 @@ private void UpdateForOrientation(Orientation orientationValue)
                     break;
             }
         }
+
+        private void UpdateInlines()
+        {
+            if (_textContent == null || Inlines == null || Inlines.Count == 0)
+            {
+                return;
+            }
+
+            _textContent.Inlines.Clear();
+            var inlines = Inlines.ToList();
+
+            Inlines.Clear();
```

Comment:
I get you want to copy the inlines over to the textcontent, but should this really get cleared out? If you inspect the inlines after this point it'll look like inlines are suddenly null?

Reply:
Yea the problem is following:
If I write the XAML like in the sample above `Inlines` gets accessed before `OnApplyTemplate`. Therefore I do not have the information about `_textContent`. Therefore I created a `new TextBlock().Inlines`. 
Now if I omit LN91 an exception indicating that the UIElement is already attached to another element is raised.
Do you have any suggestions how to hack that?

Reply:
@dotMorten Because of the comment from @nmetulev this awful hack is not needed anymore and threfore I have removed this weird code in commit fef11fd0554a5682d232af85491b50355f1f76a8